payments/card/other-features.md

```diff
@@ -725,7 +725,7 @@ sequenceDiagram
   note left of Payer: redirect back to merchant<br>(If Redirect scenario)
 
   Payer->>+Merchant: access merchant page
-  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<paymentorder.id>
+  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<payment.id>
```

Comment:
As @NullableInt writes, `payment.id` is the full URL. Just remove the path and the use of `payment.id` will be correct.

```suggestion
  Merchant->>+SwedbankPay: <payment.id>
```